akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -431,13 +434,14 @@ private final class Sdk(
       { context =>
 
         val workflow = wiredInstance(clz) {
-          sideEffectingComponentInjects(None).orElse {
+          sideEffectingComponentInjects(context.asInstanceOf[WorkflowContextImpl].openTelemetrySpan).orElse {
```

Comment:
Is there a good reason to not have the `context` be required to be a `WorkflowContextImpl` all over this impl class?

Reply:
nope, changing